### PR TITLE
Pipelines Docs Cleanup

### PR DESCRIPTION
**Pull request by geriom**
# Changes

- Update weights to group similar docs on website
- Add missing front matter to docs where it's missing
- Delete obsolete docs

@afrittoli classified the current Pipelines documentation and reported issues with some documents missing the front matter, these docs show up as blank spaces in the left nav at https://tekton.dev/docs/pipelines/

[This spreadsheet](https://docs.google.com/spreadsheets/d/1HknnR5FDSPgLJlI7kbtPsUlvL1nGvYeIZA7vKti05os/edit?usp=sharing) has the full list of docs, the classifications, those marked to be deleted, and the new suggested order.

The new categories, by doc `weight`, are:

- 100s: Tekton usage docs 
- 200s: Tekton Resources
- 300s: Tekton Features
- 400s: References and API Features

/kind documentation

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

**Comment by vdemeester** [comment]
> tide — Not mergeable. PR can't be rebased
This is weird, seems like it's the case for all our PRs... (cc @abayer @afrittoli )

File: docs/bundle-resolver.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<!--
+---
+linkTitle: "Bundles Reesolver"
```
Comment (AlanGreene) [inline docs/bundle-resolver.md:3]:
```suggestion
linkTitle: "Bundles Resolver"
```

File: docs/trusted-resources.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+<!--
+---
+linkTitle: "Trusted Sources"
```
Comment (AlanGreene) [inline docs/trusted-resources.md:3]: Should this be `Trusted Resources`?

**Review by AlanGreene — CHANGES_REQUESTED** [review]
Thanks @geriom, the new order makes a lot more sense to me 👍 

Small typo, otherwise looks good.

File: docs/bundle-resolver.md (hunk 1 above)
Comment (geriom) [reply docs/bundle-resolver.md:3]: fixed

File: docs/trusted-resources.md (hunk 2 above)
Comment (geriom) [reply docs/trusted-resources.md:3]: fixed

**Review by jerop — COMMENTED** [review]
/lgtm

**Pull request merged**